tests/test_embed/test_interpreter.cpp

```diff
@@ -75,6 +75,13 @@ PYBIND11_EMBEDDED_MODULE(throw_error_already_set, ) {
     d["missing"].cast<py::object>();
 }
 
+TEST_CASE("PYTHONPATH is used to update sys.path") {
+    // The setup for this TEST_CASE is in catch.cpp!
+    auto sys_path = py::module_::import("sys").attr("path").attr("__str__")().cast<std::string>();
```

Comment:
This is equivalent to the following (which is a bit more Pythonic).
```suggestion
    auto sys_path = py::str(py::module_::import("sys").attr("path")).cast<std::string>();
```